pkg/dock/discovery/discovery.go

```diff
@@ -175,6 +183,22 @@ func (pdd *provisionDockDiscoverer) Discover() error {
 
 		pdd.pols = append(pdd.pols, pols...)
 	}
+	ctx := c.NewAdminContext()
+	polsInDb, err = pdd.c.ListPools(ctx)
+	if err != nil {
+		return fmt.Errorf("Can not read pools in db")
```

Comment:
Start the message with lower letter.

Other thing looks ok to me

Reply:
Done